app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true %>
```

Comment:
This is the user log-in, not registration, but you're right, this is definitely a bug.  I think that when I used devise to generate new views for registration and log-in, the views automatically included email but not username.  I fixed it in registration but not log-in.  I'll fix this now.  (Thanks for all the feedback!) 👍  :octocat: 👍